tests/test_view.py

```diff
@@ -0,0 +1,178 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.forms import DriverSearchForm, CarSearchForm
+from taxi.models import Manufacturer, Car, Driver
+
+
+class TestCarSearch(TestCase):
+    def setUp(self):
+        self.manufacturer = Manufacturer.objects.create(
+            name="test_manufacturer", country="test_country"
+        )
+        self.car1 = Car.objects.create(
+            model="car1_test", manufacturer=self.manufacturer
+        )
+        self.car2 = Car.objects.create(
+            model="car2_test", manufacturer=self.manufacturer
+        )
+        self.car3 = Car.objects.create(
+            model="CAR1_test", manufacturer=self.manufacturer
+        )
+
+    def test_search_car_by_model(self):
+        form_data = {"model": "car1"}
+        form = CarSearchForm(data=form_data)
+        self.assertTrue(form.is_valid())
+        queryset = Car.objects.filter(
+            model__icontains=form.cleaned_data["model"]
+        )
+
+        self.assertIn(self.car1, queryset)
+        self.assertNotIn(self.car2, queryset)
+        self.assertIn(self.car3, queryset)

```

Comment:
what do u test here, u are getting filtered queryset, and then just check if car exist in filtered queryset - does this check ur search feature ?
the same is for other tests